code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm

```diff
@@ -0,0 +1,578 @@
+//Mostly copypaste from Nothing There. I promise this will look different later - Coxswain
+/mob/living/simple_animal/hostile/abnormality/nobody_is
+	name = "Nobody Is"
+	desc = "A mirror embedded in gross pink flesh."
+	health = 3000
+	maxHealth = 3000
+	attack_verb_continuous = "slashes"
+	attack_verb_simple = "slash"
+	attack_sound = 'sound/weapons/slash.ogg'
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "nobody"
+	icon_living = "nobody"
+	icon_dead = "nobody_dead"
+	melee_damage_type = BLACK_DAMAGE
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.3, PALE_DAMAGE = 1.2)
```

Comment:
```suggestion
	damage_coeff = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.3, PALE_DAMAGE = 1.2)
```

Reply:
Why? No other abnormality has brute left unspecified in their default damage coeffs

Reply:
because we literally don't need to anymore. If a value isn't specified it's defaulted to 1.